src/components/LoginComponent.jsx

```diff
@@ -26,6 +26,7 @@ class LoginComponent extends React.Component {
         localStorage.setItem('art-prod-web-token', idToken);
         this.props.history.push('/dashboard');
       });
+      ToastMessage.success({ message: 'Welcome to ART' });
```

Comment:
This message is unnecessary. The user will be directed to the dashboard so they'll know that they are already logged in